README.md

```diff
@@ -94,66 +92,62 @@ params   = { object_purpose: 'PURCHASE',
-#                "messages" => [ ] # ommitted for brevity,
-#              "rates_list" => [ ] # ommitted for brevity.
+#   "carrier_accounts"    => [],
+#   "address_from"        => [ ... ],   # omitted for brevity
+#   "address_to"          => [ ... ],
+#   "address_return"      => [ ... ],
+#   "parcel"              => [ ... ],
+#   "shipment_date"       => "2016-07-06T20:33:02.211Z",
+#   "customs_declaration" => nil,
+#   "extra"               => {
+#     "insurance"   => {
+#       "amount"    => 10,
+#       "currency"  => "USD",
+#       "content"   => "",
+#       "provider"  => "FEDEX"
+#     },
+#     "is_return"   => nil,
+#     "reference_1" => "",
+#     "reference_2" => "",
+#   },
+#   "messages"            => [ ... ],
+#   "rates"               => [ ... ]
 # }
 ```
 
 #### List Handling
 
 In the case when the API returns a hash with one of the hash values being an array of entities, and if the corresponding key can be mapped into one of the existing API models, then each of the members of the array is coerced from a hash into an object of the model's type.
 
-In the example below we are showing the result of such transformation where the `rates_list` contains a list of fully constructed objects of type `Shippo::Rate` after being coerced from a hash.
+In the example below we are showing the result of such transformation where `rates` contains a list of fully constructed objects of type `Shippo::Rate` after being coerced from a hash.
 
 ```ruby
-ap @shipment.rates_list.first
+ap @shipment.rates.first
 # =>
-# {
-#                     "shipment" => "20f25e44b16b4051b6dd910cb66fd27b",
-#             "available_shippo" => true,
-#                   "attributes" => [],
-#                       "amount" => "8.51",
-#                     "currency" => "USD",
-#                 "amount_local" => "8.51",
-#               "currency_local" => "USD",
-#                     "provider" => "FedEx",
-#            "provider_image_75" => "https://shippo-static.s3.amazonaws.com/providers/75/FedEx.png",
-#           "provider_image_200" => "https://shippo-static.s3.amazonaws.com/providers/200/FedEx.png",
-#            "servicelevel_name" => "Ground",
-#           "servicelevel_token" => "fedex_ground",
-#           "servicelevel_terms" => "",
-#                         "days" => 2,
-#                   "arrives_by" => nil,
-#               "duration_terms" => "",
-#                    "trackable" => true,
-#                    "insurance" => false,
-#       "insurance_amount_local" => "0.00",
-#     "insurance_currency_local" => nil,
-#             "insurance_amount" => "0.00",
-#           "insurance_currency" => nil,
-#            "delivery_attempts" => nil,
-#            "outbound_endpoint" => "door",
-#             "inbound_endpoint" => "door",
-#                     "messages" => [],
-#              "carrier_account" => "4b1940bc69524163b669asd361842db",
-#                         "test" => true
+# { 
```

Comment:
all that changed is 1. styling so it's more readable imo, and 2. fields for v3 were changed according to the confluence page